electron/anticheat/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { heroicAnticheatDataPath, isLinux } from '../constants'
+import * as axios from 'axios'
+import { logInfo, LogPrefix, logWarning } from '../logger/logger'
+import { readFileSync, writeFileSync } from 'graceful-fs'
+import { AntiCheatInfo } from '../types'
+import { epicTitleToAnticheat } from './titles_map'
+
+async function downloadAntiCheatData() {
+  if (!isLinux) return
+
+  try {
+    const { data } = await axios.default.get(
+      'https://raw.githubusercontent.com/Starz0r/AreWeAntiCheatYet/master/games.json'
```

Comment:
```suggestion
      'https://raw.githubusercontent.com/Starz0r/AreWeAntiCheatYet/HEAD/games.json'
```

To avoid branch renaming from breaking it.